inc/RestCommand.php

```diff
@@ -61,9 +61,15 @@ public function set_auth( $auth ) {
 	public function create_item( $args, $assoc_args ) {
 		list( $status, $body ) = $this->do_request( 'POST', $this->get_base_route(), $assoc_args );
 		if ( Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
-			WP_CLI::line( $body['id'] );
+			if ( $status < 400 ) {
```

Comment:
Caveat, I am learning Travis on the fly. Might see several commits as I figure out how to use it.
I found examples where the REST api returns list results with no "id". I wasn't able to reverse-out the spec on what to do in those situations. So in this case, what *is* the spec/rule? Here's the scenario
- user types command line to create an item
- code calls a REST api to create item. code expects an id to be returned
- code receives an http result code < 400 yet no id
What is the output for the case with and without porcelain?

It is possible for REST to return http=200, a body, and no id. So that means the server did...something. I intentionally left the code in a manner that errors will occur so that the admin can see something unexpected happened. Think of porcelain, in that usage I would output nothing. A user then knows the create failed yet has no diagnostics yet the server did something. I didn't like that scenario so I was allowing the code to fault so it could provide diagnostics.
But...what do you think. What is the approach/spec on this? For both porcelain and without?